src/Cargo.lock

```diff
@@ -2752,15 +2775,16 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 "checksum rls-vfs 0.4.4 (registry+https://github.com/rust-lang/crates.io-index)" = "ffd34691a510938bb67fe0444fb363103c73ffb31c121d1e16bc92d8945ea8ff"
 "checksum rustc-demangle 0.1.5 (registry+https://github.com/rust-lang/crates.io-index)" = "aee45432acc62f7b9a108cc054142dac51f979e69e71ddce7d6fc7adf29e817e"
 "checksum rustc-serialize 0.3.24 (registry+https://github.com/rust-lang/crates.io-index)" = "dcf128d1287d2ea9d80910b5f1120d0b8eede3fbf1abe91c40d39ea7d51e6fda"
+"checksum rustfmt-nightly 0.2.16 (registry+https://github.com/rust-lang/crates.io-index)" = "8cb7fea6f3c17082c41911b1de35e6585ebbf01f08435f07014f0327b56527b5"
```

Comment:
I feel like maybe something didn't quite resolve correctly here? We're depending on two versions of rustfmt with this -- one in-tree (I think) and one on crates.io. Technically not a problem, but maybe worth looking into?